book/page_creation.rst

```diff
@@ -566,9 +566,9 @@ Ok, time to finish mastering the fundamentals by reading these chapters:
 * :doc:`/book/routing`
 * :doc:`/book/templating`
 
-Then, learn about Symfony's :doc:`service container </book/service_container>`
+Then, learn about Symfony's :doc:`service container </book/service_container>`,
 the :doc:`form system </book/forms>`, using :doc:`Doctrine </book/doctrine>`
-if you need to query a database and more. in the :doc:`Symfony Book </book/index>`.
+if you need to query a database and more in the :doc:`Symfony Book </book/index>`.
```

Comment:
@WouterJ Yes, me too. Maybe it would be more readable by putting it like that? :

```
Then, in the Symfony Book, learn about the service container, the form system,
using Doctrine (if you need to query a database), and more!
```